python/eos/plot/plotter.py

```diff
@@ -374,80 +361,75 @@ class Observable(BasePlot):
            }
         """)
 
+        from .specs import Specs, ParametersSpec, KinematicsSpec,\
+        RangeSpec, ObservableSpec, VariableSpec, ParameterFileSpec
+
+        specs_prototype = Specs([
+            ObservableSpec(name='observable'),
+            VariableSpec(name='variable'),
```

Comment:
Why is there not ``RangeSpec``, which is mandatory for a plot item of type ``observable``?

Reply:
It's not implemented in the current code, but that's fixed easily.